tests/integration_tests/test_servercontext.py

```diff
@@ -70,36 +70,6 @@ def fin():
     mock_server.on_connection_lost.assert_any_call()
 
 
-async def test_broadcast_raw(context, mock_server):
-    srv, ctx = context
-    (reader, writer) = await asyncio.open_connection(
-        *srv.sockets[0].getsockname()
-    )
-    writer.close()
-
-    # If connection errors aren't handled, this should fail due to a
-    # ConnectionError
-    for _ in range(20):
-        await ctx.broadcast_raw(b"Some bytes")
-
-    assert len(ctx.connections) == 0
-
-
-async def test_broadcast(context, mock_server):
-    srv, ctx = context
-    (reader, writer) = await asyncio.open_connection(
-        *srv.sockets[0].getsockname()
-    )
-    writer.close()
-
-    # If connection errors aren't handled, this should fail due to a
-    # ConnectionError
-    for _ in range(20):
-        await ctx.broadcast(["Some message"])
-
-    assert len(ctx.connections) == 0
-
-
```

Comment:
I'm offering to add a little bit performance test, create like 100 clients, one server instance and profile it before and after

Reply:
How would you determine if the test failed or succeeded? It would also be dependent on the hardware it's running on, and any constraints we would try to put on it would have to be pretty low so it would work on travis. 

Another problem is that the profile data depends largely on the clients and what they are doing. So in order to get a semi accurate profile data we would have to simulate the clients in an accurate way as if they were real clients. And that's another rabbit hole which Brutus and I have tried to go down already, but it would take a lot of work to get something which would simulate even remotely accurate usage data.